src/backend/api/meals.js

```diff
@@ -1,8 +1,152 @@
+const { query } = require("express");
 const express = require("express");
+const { max } = require("../database");
 const router = express.Router();
 const knex = require("../database");
 
 router.get("/", async (req, res) => {
+<<<<<<< HEAD
```

Comment:
Looks like you tried to resolve a merge conflict and forgot to remove this line and also some others further down in the file. Because of this, it doesn't look like this file can run at this point. Can you look into getting that sorted out?

Reply:
I have updated conflict error. Thank you for awesome feedback.